sbt-scala-native/src/main/scala/scala/scalanative/sbtplugin/ScalaNativePluginInternal.scala

```diff
@@ -322,6 +322,16 @@ object ScalaNativePluginInternal {
       IO.read(nativeNativelib.value / "target")
     },
     nativeMode := "debug",
+    nativeDebug := {
+      val binary = abs(nativeLink.value)
+      val lldb   = abs(nativeLLDB.value)
+      val logger = streams.value.log
+      Process(lldb +: Seq("-f", binary, "-o", "run"))
+        .run(logger, connectInput = true)
+        .exitValue
+
+      ()
+    },
```

Comment:
It doesn't really matter as that flag only applies for the C/C++ code, not LLVM that we generate. We don't emit any debug symbols ourselves yet (see #29.)